stankove_napady.md

```diff
@@ -0,0 +1,105 @@
+
+> zvysit povedomi o PyLadies celkove (stale se setkavam jako organizatorka s tim, ze spousty python lidi o teto aktivite lidi z komunity vubec nevi)
+
+> zvysit povedomi i u mensich firem, ktere by to jakkoliv rady podporily
+
+> moznost jak zapojit zase nove kouce, kteri by napriklad opravdu radi Python ucili o moznosti na PyLadies nevi
+
+**naucse.python.cz**
+
+> otazka na @encukou  zdali jsme schopni za mesic posunou tuto novou vec tak, abychom ji tam mohli prezentovat?
+
+> vidim to jako velmi zajimave doplneni stanku vedle PyLadies - bude pekne videt, ze mame zajem ucit vsechny bez rozdilu
+
+**Zapojse a 📚  blog.python.cz**
+
+> to by tam nemelo chybet
+
+**PyCon?**
+> ze bude?
+
+<br>
+</br>
+## Kdo to muze prezentovat - sonda do vaseho casoveho rozvrhu
+je mi velmi jasne, ze vetsina z nas rada uslysi prednasky. Az tedy na me, ja se na ne tolik nehrnu. Bylo by lepsi dat si dopredu jak se da s vami na stanku pocitat ci zdali bych si s sebou mela nekoho vzit z holek - ale nemyslim si, ze je tolik dobry napad, aby tam byly jen holky - mozna by mel zajem @stlk?
+
+| Kdo        | Kdy           | Jak muzu pomoci |
+| ------------- |:-------------:| -----:|
+| Veronika    | klidne celou dobu | muzu povidat skoro o vsem - nemam ale tolik co rict k zapojse a naucse |
+|    ?   |       |    |
+|   ?    |       |    |
+
+<br>
+</br>
+## Co by na stanku nemelo chybet
+
+> v prvni rade, dotaz organizatoru je, zdali potrebujeme LAN  (wifi jinak k dispozici bude) a zdali potrebujeme zasuvky, toto je zrejme otazka na @encukou, zatim ji neumim sama rozhodnout
+<br>
+</br>
+**1. fotky z akci vytistene, hlavne hipsta fotky s ESPckem a hipsa s UKULELE** @lspdv zaridi slozku, kde vybereme, necha vytisknout, meli bychom na to mit k dispozici nejake nastenky, dohodnu v organizatorama
+
+**2. video z Pyconu nebo videa z prednasek z Pyva** @encukou a dulezite pro @lspdv vedet na cem to chceme spoustet
+
+**3. samolepy PyLadies** @lspdv zaridi tisk pr. tiskovou pripravu
+
+> navrhuji udelat podstatne mensi a jen cervene bez hlav Máni
+
+> zvazuji udelat navrh malych, kde bude napis pyladies a neco ve stylu "I support girls who love to code in Python - touzim po tom, aby lidi vedeli ze samolepky o co jde, ciste pyladies je hrozne nudo
+
+**4. samolepky python.cz** @honzajavorek navrh, @lspdv zaridi tisk pr. tiskovou pripravu
+
+> napad prvni (@lspdv), mensi bile kulate (prumer 3cm?), kde by byla nejaka uplne nova maluvka (neco s pythonem jak sedi u kompu kouka na python.cz) nebo maluvka jak je python obtoceny kolem zidle, nebo maluvka kde pythonista v kocaru pohanenym pythony(ta je na FB) a pod to jen python.cz 
+
+> druhy @lspdv napad byl opet mensi bile kulate - ale na tema blog.python.cz 
+
+_nerikam nutne oboji - muzeme toho vyuzit k propragaci toho co potrebuje propagovat vic. Podle me to je python.cz a tam se najde vse dulezite dal_
+
+**5. letacek velikosti A5 jednostranny/ obostranny** @lspdv navrh, @honzajavorek muze propujcit jiz hotove maluvky?
```

Comment:
klidne muzu ilustrovat i novyma maluvkama, ale chtelo by to strucnej obsah mit uz predem nez pro to budu neco kreslit